akka-ssl-config/src/main/scala/com/typesafe/sslconfig/akka/AkkaSSLConfig.scala

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+
+package com.typesafe.sslconfig.akka
```

Comment:
I think I'm missing the background for this library. So there is already something called ssl-config-akka published that is using exactly this package name?
I'm afraid of problems that will occur with transitive dependencies including both ssl-config-akka and this akka-ssl-config, and they are using same package/class names.